jdbc/src/main/java/com/interface21/jdbc/exception/JdbcAccessException.java

```diff
@@ -0,0 +1,11 @@
+package com.interface21.jdbc.exception;
+
+public class JdbcAccessException extends RuntimeException {
+    public JdbcAccessException(String message, Throwable cause) {
+        super(message, cause);
+    }
```

Comment:
```suggestion
public class JdbcAccessException extends RuntimeException {

    public JdbcAccessException(String message, Throwable cause) {
        super(message, cause);
    }
```

다른 클래스에서는 개행을 하고 계시네요!!